static/main.css

```diff
@@ -668,34 +667,303 @@ header .header-list.open li img {
   background-position: 0%;
   transition: all 300ms ease-in-out;
 }
-
 .movie-details .movie-showtimes .movie-showtimes-list .movie-tickets:hover {
   background-position: -200%;
   transition: all 300ms ease-in-out;
 }
 
-.open {
-  width: 75%;
-  margin-inline: 30%;
+.noResult {
+  display: none;
+  font-size: 1.5rem;
+  justify-content: center;
 }
-.open .opentimes, .open .contact {
-  list-style-type: none;
+
+#search-icon {
+  color: rgb(0, 0, 0);
+  font-size: 1.3em;
 }
-.open .rent {
-  list-style-type: georgian;
+
+.search-bx .search {
+  align-items: center;
+  color: #161a1d;
+  display: flex;
+  position: relative;
 }
-.open li, .open p {
+.search-bx .search input {
+  background: transparent;
+  border-radius: 20px;
+  border: none;
+  color: #161a1d;
+  outline: none;
+  transition: 1s linear;
+  -webkit-transition: 1s linear;
+  width: 0px;
+}
+.search-bx .search i {
+  color: rgba(255, 255, 255, 0.7);
+  cursor: pointer;
+  font-size: 13px;
+  transition: 0.4s linear;
+}
+.search-bx .search .search-bx2 {
+  background: rgba(22, 26, 29, 0.0823529412);
+  border-radius: 10px;
+  display: block;
+  height: 300px;
+  left: -10px;
+  opacity: 0;
+  overflow: auto;
+  position: absolute;
+  scrollbar-width: thin;
+  scrollbar-color: rgba(22, 26, 29, 0.0823529412) rgba(22, 26, 29, 0.0823529412);
+  top: 35px;
+  transition: 0.3s linear;
+  visibility: hidden;
+  width: 280px;
+}
+.search-bx .search .search-bx2 a {
+  align-items: center;
+  color: #161a1d;
+  display: flex;
+  height: 100px;
+  text-decoration: none;
+  transition: 0.3s linear;
+  width: 100%;
+}
+.search-bx .search .search-bx2 a img {
+  height: 100%;
+  padding-right: 5px;
+  top: 0px;
+  width: 80px;
+}
+.search-bx .search .search-bx2 a .search-content {
+  color: #161a1d;
+}
+.search-bx .search .search-bx2 a .search-content h6 {
+  font-size: 13px;
+}
+.search-bx .search .search-bx2 a .search-content p {
+  color: #161a1d;
+  font-size: 11px;
+  font-weight: 500;
+}
+.search-bx .search .search-bx2 a:hover {
+  background: #edf2f4;
+}
+.search-bx .search .search-input {
+  padding: 3px 0px 3px 8px;
+  width: 280px;
 }
 
-.about {
-  width: 75%;
-  margin-inline: 30%;
+@media (max-width: 500px) {
```

Comment:
in variables.scss we have the variable :
$bp-small: 500px;
that would be better if you use that here.

Reply:
Sorry, my mistake , this comment was not relevent to your PR :(